main.go

```diff
@@ -136,6 +139,8 @@ func main() {
 	var wrappedHandler http.Handler = handler
 	if len(opts.MetricsListenAddr) > 0 && len(strings.Split(opts.MetricsListenAddr, ":")) == 2 {
 		metricsHandler := http.NewServeMux()
+		//add health on metrics http to serve k8s liveness
+        metricsHandler.HandleFunc("/health",health)
```

Comment:
unrelated changes - please send as separate PR - and address indentation + formatting.